statsmodels/tools/tools.py

```diff
@@ -18,6 +20,10 @@ def asstr2(s):
         return str(s)
 
 
+def is_wasm():
```

Comment:
this function should be in compat/python.py since it is for specific python version compatability.   I think you should also define it as a constant, `IS_WASM` or similar, which is more the style of the checks in `@skip` or `@xfail` decorators (e.g., `PANDAS_LT_3`).

Reply:
Addressed in bf68d552db092c33a92df20ac18c4a468e8f381c.